### DISCUSSION TITLE
Automatically create incidents utilising the observability tools

### DISCUSSION THREAD

**Discussion by spirosoik** [mattermost/community#11] [Moonshot Ideas]
Machine Learning model for anomaly detection based on the observability metrics (monitoring metrics, distributed tracing, alerts, logging) and raise automatically an incident to be assessed and approved by the team as a real incident or not.

So the idea is automatically create incidents for application services, infrastructure etc. utilising the observability tools. The incident should include all the info which used to detect the incident like:
- which metrics or a grafana view given a specific time which have been violated
- logging errors at the specific time
- alerts 
Etc. etc.

**Comment by chetanyakan** [comment]
**Fun idea!**
Could also be interesting to monitor fraudulent activity for system admins (and potentially other users) for their accounts getting "hacked".
